checkov/terraform/checks/resource/ncp/LBNetworkPrivate.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class LBNetworkPrivate(BaseResourceCheck):
+
+    def __init__(self):
+        name = "Ensure LB isn't exposed to the internet"
```

Comment:
```suggestion
        name = "Ensure Load Balancer isn't exposed to the internet"
```
you can write out LB 🙂